### PR TITLE
Isolate global dataLayer

### PR DESCRIPTION
**Pull request by aetheon**
Hi there,

There's an issue where window.dataLayer can be used by multiple google tag manager usages within a page. This PR adds the possibility to specify `globalDataLayerName` to achieve isolation.

### CODE REVIEW

**Comment by MatteoGabriele** [comment]
the tests are not passing, not sure if you want to finish the PR or not, otherwise i will close it

**Comment by aetheon** [comment]
Was just seeing if this was being actively maintained. Will fix the tests tomorrow. Tks

**Comment by aetheon** [comment]
hi @MatteoGabriele updated the PR with latest and fixes CI.

**Comment by MatteoGabriele** [comment]
> the tests are not passing, not sure if you want to finish the PR or not, otherwise i will close it

it is, but when I am not too fried after a day of work 🤣

**Comment by MatteoGabriele** [comment]
> Was just seeing if this was being actively maintained. Will fix the tests tomorrow. Tks

it is, but when I am not too fried after a day of work 🤣

File: src/bootstrap.js [hunk 1]
```diff
@@ -26,9 +27,9 @@ export default function() {
   }
 
   if (window[globalObjectName] == null) {
-    window.dataLayer = window.dataLayer || [];
+    window[globalDataLayerName] = [];
```
Comment (MatteoGabriele) [inline src/bootstrap.js:30]: this one should be `window[globalDataLayerName] = window[globalDataLayerName] || []`

**Comment by aetheon** [comment]
@MatteoGabriele applied that change tks.

**Comment by aetheon** [comment]
tks 👍

**Comment by MatteoGabriele** [comment]
Thanks to you for the feature! 
Cheers

**Pull request merged**